src/instance.ts

```diff
@@ -192,6 +238,249 @@ class Instance extends common.GrpcServiceObject {
+   *
+   * @param {string} backupId The name of the backup.
+   * @param {string} databasePath the path of the backup.  Only needs to be specified for new backups.
+   * @param {PreciseDate} expireTime expiry time of the backup.  Only needs to be specified for new backups.
+   * @return {Backup} A Backup object.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const backupExpiryDate = new PreciseDate(Date.now() + 1000 * 60 * 60 * 24)
+   * const backup = instance.backup('my-backup', 'projects/my-project/instances/my-instance/databases/my-database', backupExpiryDate);
+   */
+  backup(
+    backupId: string,
+    databasePath: string,
+    expireTime: PreciseDate
+  ): Backup {
+    if (!backupId) {
+      throw new Error('A backup ID is required to create a backup.');
+    }
+
+    return new Backup(this, backupId, databasePath, expireTime);
+  }
+
+  getBackups(query?: GetBackupsRequest): Promise<GetBackupsResponse>;
+  getBackups(callback: GetBackupsCallback): void;
+  getBackups(query: GetBackupsRequest, callback: GetBackupsCallback): void;
+  /**
+   * Query object for listing backups.
+   *
+   * @typedef {object} GetBackupsRequest
+   * @property {boolean} [autoPaginate=true] Have pagination handled
+   *     automatically.
+   * @property {number} [maxApiCalls] Maximum number of API calls to make.
+   * @property {number} [maxResults] Maximum number of items to return.
+   * @property {number} [pageSize] Maximum number of results per page.
+   * @property {string} [pageToken] A previously-returned page token
+   *     representing part of the larger set of results to view.
+   */
+  /**
+   * @typedef {array} GetBackupsResponse
+   * @property {Backup[]} 0 Array of {@link Backup} instances.
+   * @property {object} 1 The full API response.
+   */
+  /**
+   * List backups on the instance.
+   *
+   * Both completed and in-progress backups are listed if no filter is supplied.
+   *
+   * @see {@link #backup}
+   *
+   * @param query query object for listing backups.
+   * @returns {Promise<GetBackupsResponse>} when resolved, contains a paged list of backups.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const [backups] = await instance.getBackups();
+   */
+  getBackups(
+    queryOrCallback?: GetBackupsRequest | GetBackupsCallback,
+    cb?: GetBackupsCallback
+  ): void | Promise<GetBackupsResponse> {
+    const callback =
+      typeof queryOrCallback === 'function' ? queryOrCallback : cb!;
+    const query =
+      typeof queryOrCallback === 'object'
+        ? queryOrCallback
+        : ({} as GetBackupsRequest);
+
+    const reqOpts = extend({}, query, {
+      parent: this.formattedName_,
+    });
+    this.request<IBackup[]>(
+      {
+        client: 'DatabaseAdminClient',
```

Comment:
This seems misplaced. Since a backup is connected to a database, it seems like the Database class would be the place to put this. It is handy to get a more global collection by keeping this on Instance as well, so just adding this method on the Database class could work, too.

Reply:
I understand what you mean. But there is a good reason for putting it in the Instance rather than Database. Once a backup is created, it is no longer connected to a database. The backup resource is part of an instance (i.e. /project/my-project/instance/my-instance/backup/my-backup). The underlying database the backup came from could be deleted but the backup will still exist. So the backend team asked us to make this method part of the Instance despite the fact that the RPC is in the database client.

Reply:
That makes sense. Thanks for taking the time to help me understand!